content/events/2020/trivia/contents.lr

```diff
@@ -0,0 +1,16 @@
+title: Trivia Day
+---
+body:
+
+# TRIVIA DAY
+We are having a trivia day. The trivia will be done over a 24-hour period to ensure that people in all time zones can participate. Points will be awarded on correct answers and can be done unlimited times. 
+
+#### Thursday, November 5th
+##Want to participate? Just come on down to [DISCORD](https://discord.gg/hgmcNKP2)!
```

Comment:
```suggestion
## Want to participate? Just come on down to [DISCORD](http://discord.muncompsci.ca/)!
```

http://discord.muncompsci.ca/ is the proper link for discord, we can update what this points to if the invite link expires.